### DISCUSSION TITLE
Off topic fun - Discussions that aren’t really major discussions but are still discussions :-)

### DISCUSSION THREAD

**Discussion by lcn2** [ioccc-src/winner#26] [General]
This new discussion is a continuation of [issue 5 from the temp-test-ioccc repo](https://github.com/ioccc-src/temp-test-ioccc/issues/5) that was originally started by @xexyl.

This discussion is for people to point j-random and perhaps somewhat off topic comments.

Please keep this discussion polite, friendly, reasonably clean and fun 🤩

-- Thank You.

**Comment by lcn2** [comment]
The word of tomorrow is [fizzbin](https://en.wikipedia.org/wiki/List_of_games_in_Star_Trek#Fizzbin).  🤓

**Reply by SirWumpus** [reply]
Only if its a Tuesday.

**Reply by xexyl** [reply]
> Only if its a Tuesday.

What if it's Wacky Wednesday?

**Reply by SirWumpus** [reply]
Then put on a pretty floral bonnet and Brown Coat, walk down the High Street to the nearest Pub, and ask for a `Gin Fizz`.

**Reply by xexyl** [reply]
> Then put on a pretty floral bonnet and Brown Coat, walk down the High Street to the nearest Pub, and ask for a `Gin Fizz`.

![freepik__candid-image-photography-natural-textures-highly-r__11124](https://github.com/user-attachments/assets/481ad87a-102a-44da-b50b-ed1abf2c1413)

**Reply by SirWumpus** [reply]
`pretty floral bonnet and Brown Coat,` [Firefly](https://www.imdb.com/title/tt0303461/) references.

**Reply by xexyl** [reply]
> `pretty floral bonnet and Brown Coat,` [Firefly](https://www.imdb.com/title/tt0303461/) references.

Oh ... Alan Tudyk is in it. That should be fun! I tried to modify the image above (well new prompt) but it didn't turn out like I wanted and I don't want to waste more for a silly thing (as in my reaction is silly) so I won't bother. I could have got a transparent picture of a firefly and put it over her head instead but ... anyway, you get the idea.

**Comment by lcn2** [comment]
We mentioned it in [GH-discussioncomment-11700162](https://github.com/ioccc-src/winner/discussions/29#discussioncomment-11700162), but it probably is better posted here regarding the phrase "the other repo".

# "the other repo"

BTW: "the other repo" is a traditional phrase borrowed the phrase "[the other place](https://en.wikipedia.org/wiki/Another_place)" as used on the Westminster parliament when we refer to related repos that are not part of this repo, such as the [mkiocccentry repo](https://github.com/ioccc-src/mkiocccentry).

And while the phrase has its roots in antiquity based on rivalry, we use in jest and good fun. 🤩

**Reply by xexyl** [reply]
> We mentioned it in [GH-discussioncomment-11700162](https://github.com/ioccc-src/winner/discussions/29#discussioncomment-11700162), but it probably is better posted here regarding the phrase "the other repo".
> 
> # "the other repo"
> BTW: "the other repo" is a traditional phrase borrowed the phrase "[the other place](https://en.wikipedia.org/wiki/Another_place)" as used on the Westminster parliament when we refer to related repos that are not part of this repo, such as the [mkiocccentry repo](https://github.com/ioccc-src/mkiocccentry).
> 
> And while the phrase has its roots in antiquity based on rivalry, we use in jest and good fun. 🤩

Interesting. I didn't have that in mind when I first said it .. and I said it before you did.  It's interesting you had something specific in mind. 

Another version that I'm sure you've encountered, maybe personally, is that some people will say, to patronise others, 'the X way'. So for instance I have said things like 'the C++ way'.

**Reply by lcn2** [reply]
Well to keep things fun, and for the record, when we used:

```
the "other repo"
```

we probably should have used:

```
"the other repo"
```

The friendly rivalry of "the other place" is and important part of this term.

The "the other place" refers to something that its **quite essential or important (just not here)** and in no way should be considered dismissive or patronizing.

**Reply by xexyl** [reply]
> Well to keep things fun, and for the record, when we used:
> 
> ```
> the "other repo"
> ```
> 
> we probably should have used:
> 
> ```
> "the other repo"
> ```
> 

I don't know if I ever quoted it but for me the placement doesn't really matter.  But fair enough.

> The friendly rivalry of "the other place" is and important part of this term.
> 
> The "the other place" refers to something that its **quite essential or important (just not here)** and in no way should be considered dismissive or patronizing.

Of course. And anyone who does think it's that **wants** it to be and therefore they should thank you for it!

**Reply by SirWumpus** [reply]
What came to mind was was "the hot place" as referenced in Firefly.

**Reply by xexyl** [reply]
> What came to mind was was "the hot place" as referenced in Firefly.

![freepik__candid-image-photography-natural-textures-highly-r__2743](https://github.com/user-attachments/assets/fd5267f0-ade7-4ae9-b92a-06511e4cf1f2)

... though at a quick tired glance that looks more like a fly ...

**Comment by xexyl** [comment]
> About Mon Dec 30 13:33:31 PST 2024 this is the last comment before this repo goes into archived / Read-Only state.
>
> Thank you to all who contributed to this wonderful repo. We hope you enjoyed your time working with this repo. Your efforts ARE MOST APPRECIATED, not only by @lcn2 but the many who now enjoy the merge target of this repo.

I certainly did enjoy it! It was an honour and privilege. The stories told, the jokes told (and meme and other fun and funny pictures being shared), the Tolkien stuff (to the depths that most people don't even know about let alone could manage), the things you taught me (and things I taught you): all worth it and more!  Plus things I learnt just working on it (or related things). It was all a delight! I'm glad we're past it but only because it means we're at the next step. It's something I'll remember <del>for the rest of my life</del> until I get dementia (which I hope never happens and I do things that should help prevent it but  who knows)!

You're most welcome and thank you to you too!

**Reply by lcn2** [reply]
FYI: The above comment refers to and quotes [GH-issuecomment-2565938094](https://github.com/ioccc-src/temp-test-ioccc/issues/2239#issuecomment-2565938094) in the now [Archived and now Read-Only temp-test-ioccc repo](https://github.com/ioccc-src/temp-test-ioccc).

**Comment by lcn2** [comment]
Wishing all of you a Happy 2025 🥳‼️

**Reply by xexyl** [reply]
> Wishing all of you a Happy 2025 🥳‼️

And to you and yours too!

**Reply by xexyl** [reply]
> Wishing all of you a Happy 2025 🥳‼️

Old meme...

![IMG_2134](https://github.com/user-attachments/assets/052675d3-f28f-4df6-a7ad-fc62a90757a0)


Now you knowing my sense of humour might think I made this but in this case I cannot take credit even if it sounds like something I would say.

**Comment by lcn2** [comment]
Here is a classic from the 1980's I think:

![image](https://github.com/user-attachments/assets/00990cce-55e3-4ff6-8ffd-826aa53a1ca0)

**Reply by xexyl** [reply]
> Here is a classic from the 1980's I think:

I thought you were going to talk about me (1982 as you know)!

.. no, I did not but I did not expect this either!

Very funny. Reminds me of other jokes out there (and probably some of my own). Thanks for the laugh!

# UPDATE 0

... oh, you had more to do .. well that ruins my joke a bit but that's okay.

**Reply by xexyl** [reply]
> Here is a classic from the 1980's I think:
> 
> ![image](https://private-user-images.githubusercontent.com/10429707/400161430-00990cce-55e3-4ff6-8ffd-826aa53a1ca0.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYwMjAxODUsIm5iZiI6MTczNjAxOTg4NSwicGF0aCI6Ii8xMDQyOTcwNy80MDAxNjE0MzAtMDA5OTBjY2UtNTVlMy00ZmY2LThmZmQtODI2YWE1M2ExY2EwLmpwZWc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEwNFQxOTQ0NDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xNTJlYmI5NzRmZTYxMDU3ZGM2YzdkZmRhMzYyYWRkNDJhOWFmZDcyYjg1ZTllOGE2MWE5ZTM0NDdhZDZiZjFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.4LEpT4g9wDNOfB8kcr8dV2vuVPtL68WCGgQs9_HFtcU)

Now that you updated it. That's brilliant! I guess you know the song 'Write In C'?

**Comment by xexyl** [comment]
Something I did a while back that some of you might find amusing or fun. Landon saw it back then:

![C-operator-table](https://github.com/user-attachments/assets/a660c9b2-8733-4e05-a92a-f923b3ce05b7)

**Comment by lcn2** [comment]
# FYI

The following is a test widget to join the IOCCC discord server.  Please note that the IOCCC discord server is **NOT** fully setup and that the initial configuration is still being worked on:

## UPDATE 0a

See below.

## UPDATE 1

[test link to join the IOCCC discord server that is still being configured](https://discord.gg/2hVYnv83DD)

**Comment by lcn2** [comment]
We believe we have addressed all of the current questions that still need answering at this time. If we've missed something or something else needs to be clarified, please ask again.
